newrelic/resource_newrelic_nrql_alert_condition.go

```diff
@@ -75,7 +83,7 @@ func resourceNewRelicNrqlAlertCondition() *schema.Resource {
 						"duration": {
 							Type:         schema.TypeInt,
 							Required:     true,
-							ValidateFunc: intInSlice([]int{1, 2, 3, 4, 5, 10, 15, 30, 60, 120}),
+							ValidateFunc: validation.IntAtLeast(1),
```

Comment:
1 is not a valid value. Must be 2-120.

Reply:
This is fixed in PR #136 which is open since may

Reply:
#136 has been closed, however #178 addressed this and has been merged.